iot/index.html.slim

```diff
@@ -5,7 +5,7 @@ description: A list of efforts that JBoss people and projects are involved with
 ---
 
 p
-  | These pages are intended to record the work that is going on within JBoss in the area of Internet of Things (IoT). At present we have no single IoT related project because it encompasses many different technologies, such as messaging and development frameworks. As such, much work on IoT is occuring within projects where that effort is more relevant, e.g., ActiveMQ/HornetQ for MQTT. You should feel free to engage these projects and their communities directly if you have an interest, questions etc. However, we also have specific forums for IoT which are available here for discussions that are not obviously relevant in one of our other projects. If you are at all unsure, use 
+  | These pages are intended to record the work that is going on within JBoss in the area of Internet of Things (IoT). At present we have no single IoT related project because it encompasses many different technologies, such as messaging and development frameworks. As such, much work on IoT is occuring within projects where that effort is more relevant, e.g., ActiveMQ/HornetQ for MQTT. You should feel free to engage these projects and their communities directly if you have an interest, questions etc. However, we also have specific forums for IoT which are available here for discussions that are not obviously relevant in one of our other projects. If you are at all unsure, use
```

Comment:
Sorry to be picky :grimacing:, but there is a space missing between 'use' and the following link 'this forum for things....'